BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/listeners/BetterShardsListener.java

```diff
@@ -370,25 +375,25 @@ public void playerSleepInBed(PlayerInteractEvent event) {
 	
 	// Start of methods to try and stop interaction when transferring.
 	
-	@EventHandler(priority = EventPriority.HIGHEST)
+	@EventHandler(priority = EventPriority.LOWEST)
 	public void playerDropItem(PlayerDropItemEvent event) {
 		Player player = event.getPlayer();
-		if(plugin.isPlayerInTransit(player.getUniqueId())) {
+		if(BetterShardsPlugin.getTransitManager().isPlayerInTransit(player.getUniqueId())) {
 			event.setCancelled(true);
 		}
 	}
 	
-	@EventHandler(priority = EventPriority.HIGHEST)
+	@EventHandler(priority = EventPriority.LOWEST)
 	public void playerDamageEvent(EntityDamageEvent event) {
-		if (!(event.getEntity() instanceof Player))
+		if (!(event.getEntity().getType() == EntityType.PLAYER))
```

Comment:
Using instanceof is never bad.

The Type isn't strictly related to the class (cast type) it's just an internal Bukkit thing fwiw. So if you know it needs to fulfill the Player interface, check `instanceof`. If you're okay with dealing with it as an Entity but also would like to note its a player, use EntityType comparison.